keepassxc-browser/content/fields.js

```diff
@@ -46,7 +46,11 @@ kpxcFields.getAllCombinations = async function(inputs) {
         }
     }
 
-    if (kpxc.singleInputEnabledForPage && combinations.length === 0 && usernameField) {
+    // Allow single input if autocomplete="username" is present, or Username-Only Detection is enabled for the page
+    const autoComplete = usernameField?.getLowerCaseAttribute('autocomplete');
+    if (combinations.length === 0 && usernameField &&
+        (kpxc.singleInputEnabledForPage || autoComplete?.includes('username') || autoComplete?.includes('email'))
```

Comment:
Do we need to lowercase the autoComplete value?

Reply:
Yes. Websites can use `Autocomplete`,  `AUTOCOMPLETE` etc. because they're not interested following the standards.

Reply:
Actually that is what the function is doing, not lowercasing the attribute name itself.. :)